### PR TITLE
ENT-8967 - Corda 4.9.6 - usekwp parameter

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/operations/deployment/hsm-deployment.md [hunk 1]
```diff
@@ -123,19 +123,24 @@ cryptoServiceName : "GEMALTO_LUNA"
 cryptoServiceConf : "gemalto.conf"
 ```
 
-The configuration file for Gemalto Luna has two fields. The `keyStore` field needs to specify a slot or partition. The `password` field contains the password associated with the slot or partition.
+The configuration file for Gemalto Luna has two paa. The `keyStore` field needs to specify a slot or partition. The `password` field contains the password associated with the slot or partition.
```
Comment (yarivus) [inline content/en/platform/corda/4.10/enterprise/operations/deployment/hsm-deployment.md:126]: is the change fields => paa intended ?

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.10/enterprise/operations/deployment/hsm-deployment.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/operations/deployment/hsm-deployment.md]: Sorry fixed typo there.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by colin-hillis — APPROVED** [review]
LGTM

**Pull request closed without merging**